source/macos/AdaptiveCards/AdaptiveCards/AdaptiveCard.swift

```diff
@@ -92,3 +94,19 @@ public struct ButtonConfig {
         self.inline = inline
     }
 }
+
+public struct HyperlinkColorConfig {
+    public static let `default` = HyperlinkColorConfig(foregroundColor: .systemBlue, isUnderlined: false, underlineColor: .blue, underlineStyle: .single)
+    
+    let foregroundColor: NSColor
+    let isUnderlined: Bool
+    let underlineColor: NSColor
+    let underlineStyle: NSUnderlineStyle
+    
+    public init(foregroundColor: NSColor, isUnderlined: Bool, underlineColor: NSColor, underlineStyle: NSUnderlineStyle) {
+        self.foregroundColor = foregroundColor
+        self.isUnderlined = isUnderlined
+        self.underlineColor = isUnderlined ? underlineColor : .clear
```

Comment:
Maybe self.underlineStyle = isUnderlined ? .single : .none
Will be better than assuming cleae color here

Reply:
> What about FactSetRenderer. Should we check there as well?
> Also, make sure all tests pass.

Added for fact set and tests pass Akaash!